packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -191,3 +192,13 @@ export function getFilteredEndowmentDescriptors(endowments: object): PropertyDes
 export function isIntrinsicGlobalName(key: PropertyKey): boolean {
     return SetHas(ESGlobalKeys, key);
 }
+
+export function setupStackTrace(global: typeof globalThis) {
+    const { Error } = global;
+    if (
+        ObjectHasOwnProperty(Error, 'stackTraceLimit') &&
+        typeof Error.stackTraceLimit === 'number'
+    ) {
+        Error.stackTraceLimit *= 2;
```

Comment:
Why double?

Reply:
It felt like a good baby step. The default is 10. So 20. Being so close to release this is a change which will help but not go overboard. I want to tread lightly.

Reply:
That's reasonable!